README.md

```diff
@@ -409,10 +410,18 @@ Style/PerlBackrefs:
 If you have a code base with an overwhelming amount of offenses, it can
 be a good idea to use `rubocop --auto-gen-config` and add an
 `inherit_from: .rubocop_todo.yml` in your `.rubocop.yml`. The generated
-file `.rubocop_todo.yml` contains configuration to disable all cops that
-currently detect an offense in the code. Then you can start removing the
-entries in the generated file one by one as you work through all the
-offenses in the code.
+file `.rubocop_todo.yml` contains configuration to disable cops that
+currently detect an offense in the code by excluding the offending
+files, or disabling the cop altogether once a file count limit has been
+reached (the default is 15 files before the cop is completely disabled).
```

Comment:
It's mentioned further down that the default is 15, so remove the text in parentheses.